### ISSUE TITLE
Doesn't work with windows 64 apparently

### ISSUE THREAD

**Issue by feriousdesteny** [MishDotCom/Admin-Password-Cracker#3]
![Screenshot 2022-11-15 135330](https://user-images.githubusercontent.com/78489481/202002036-6f564210-54a8-4c89-86be-b4824938dc61.png)

**Comment by RealVividlee** [comment]
Yeah, there is clearly a lot of context we're missing with the program. Either they're trolling (because Windows cannot natively run 16-bit applications) or we're missing other files. Seeing as both this issue and the actual post is old now, we are probably not going to get a respond.

**Comment by mistreatedbee** [comment]
where is the AdminPass.exe?
